src/main/kotlin/racingcar/CarController.kt

```diff
@@ -0,0 +1,58 @@
+package racingcar
+
+import kotlin.random.Random
+
+private const val RANDOM_RANGE = 10
+
+class CarController(private val carNames: List<String>, private val playCount: Int) {
+
+    private fun makeCars(names: List<String>): List<Car> {
+        val cars = mutableListOf<Car>()
+        for (name in names) {
+            cars.add(Car(name))
+        }
+        return cars
+    }
+
+    fun play(cars: List<Car>, accelerateNumbers: List<Int>) {
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/64#discussion_r462090513

```suggestion
    private fun play(cars: List<Car>, accelerateNumbers: List<Int>) {
```